src/modules/cpu/kernel/phase.hpp

```diff
@@ -0,0 +1,965 @@
+                    dstPtrTemp[1] = (Rpp8u) RPPPIXELCHECK(atan(((Rpp32f) (*srcPtr1TempG)) / ((Rpp32f) (*srcPtr2TempG))) * multiplier);
+                    dstPtrTemp[2] = (Rpp8u) RPPPIXELCHECK(atan(((Rpp32f) (*srcPtr1TempB)) / ((Rpp32f) (*srcPtr2TempB))) * multiplier);
+
+                    srcPtr1TempR++;
+                    srcPtr1TempG++;
+                    srcPtr1TempB++;
+                    srcPtr2TempR++;
+                    srcPtr2TempG++;
+                    srcPtr2TempB++;
+                    dstPtrTemp += 3;
+                }
+
+                srcPtr1RowR += srcDescPtr->strides.hStride;
+                srcPtr1RowG += srcDescPtr->strides.hStride;
+                srcPtr1RowB += srcDescPtr->strides.hStride;
+                srcPtr2RowR += srcDescPtr->strides.hStride;
+                srcPtr2RowG += srcDescPtr->strides.hStride;
+                srcPtr2RowB += srcDescPtr->strides.hStride;
+                dstPtrRow += dstDescPtr->strides.hStride;
+            }
+        }
+
+        // Phase without fused output-layout toggle (NHWC -> NHWC or NCHW -> NCHW)
+        else
+        {
+            Rpp32u alignedLength = bufferLength & ~15;
+
+            for(int c = 0; c < layoutParams.channelParam; c++)
+            {
+                Rpp8u *srcPtr1Row, *srcPtr2Row, *dstPtrRow;
+                srcPtr1Row = srcPtr1Channel;
+                srcPtr2Row = srcPtr2Channel;
+                dstPtrRow = dstPtrChannel;
+
+                for(int i = 0; i < roi.xywhROI.roiHeight; i++)
+                {
+                    Rpp8u *srcPtr1Temp, *srcPtr2Temp, *dstPtrTemp;
+                    srcPtr1Temp = srcPtr1Row;
+                    srcPtr2Temp = srcPtr2Row;
+                    dstPtrTemp = dstPtrRow;
+
+                    int vectorLoopCount = 0;
+                    for (; vectorLoopCount < alignedLength; vectorLoopCount += 16)
+                    {
+                        __m256 p1[2], p2[2];
+
+                        rpp_simd_load(rpp_load16_u8_to_f32_avx, srcPtr1Temp, p1);    // simd loads
+                        rpp_simd_load(rpp_load16_u8_to_f32_avx, srcPtr2Temp, p2);    // simd loads
+                        p1[0] = _mm256_round_ps(_mm256_mul_ps(atan2_ps(p1[0], p2[0]), pMul), _MM_FROUND_TO_ZERO);    // phase computation
+                        p1[1] = _mm256_round_ps(_mm256_mul_ps(atan2_ps(p1[1], p2[1]), pMul), _MM_FROUND_TO_ZERO);    // phase computation
+                        rpp_simd_store(rpp_store16_f32_to_u8_avx, dstPtrTemp, p1);    // simd stores
+
+                        srcPtr1Temp += 16;
+                        srcPtr2Temp += 16;
+                        dstPtrTemp += 16;
+                    }
+                    for (; vectorLoopCount < bufferLength; vectorLoopCount++)
+                    {
+                        *dstPtrTemp++ = (Rpp8u) RPPPIXELCHECK(atan(((Rpp32f) (*srcPtr1Temp)) / ((Rpp32f) (*srcPtr2Temp))) * multiplier);
+
+                        srcPtr1Temp++;
```

Comment:
Post increment in same line - all instances

Reply:
Same comment as above.